docs/userguide/policy/routing.md

```diff
@@ -54,7 +54,7 @@ several options.
             return kumo.make_queue_config {
                 protocol = {
                     smtp = {
-                        mx_list = { 'smart.host.local', '[10.0.0.1]' },
+                        mx_list = { 'smart.host.local', {name="mx.example.com", addr='10.0.0.1'} }
```

Comment:
Ah, the CI is unhappy about this because it hasn't been lua code formatted.  You can run `./docs/build.sh` to take care of that in the same way that it does, but really I suspect that it just wants you to use single quotes instead of double quotes around the strings